ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -173,7 +178,12 @@ case "$CLUSTER_TYPE" in
       # Generates a workspace name like rdr-mac-4-14-au-syd-n1
       # this keeps the workspace unique
       CLEAN_VERSION=$(echo "${OCP_VERSION}" | tr '.' '-')
-      WORKSPACE_NAME=rdr-mac-${CLEAN_VERSION}-${REGION}-n1
+      if [ -z "${E2E_RUN_TAGS}" ]; then
+        CUCUSHIFT="-cucushift"
+      else
+        CUCUSHIFT=""
+      fi
+      WORKSPACE_NAME=rdr-mac-${CLEAN_VERSION}-${REGION}${CUCUSHIFT}-n1
```

Comment:
Missing line https://github.com/openshift/release/blob/fe03402940619af2b07df3823ae063cc397e7ec1/ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh#L284C8-L284C8

The tag attach needs to use -cucushift or the above remove/add steps don't align.